packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -232,6 +232,12 @@ class AndroidCameraCameraX extends CameraPlatform {
     return _cameraEvents(cameraId).whereType<CameraInitializedEvent>();
   }
 
+  /// The camera's resolution has changed.
+  @override
+  Stream<CameraResolutionChangedEvent> onCameraResolutionChanged(int cameraId) {
+    return _cameraEvents(cameraId).whereType<CameraResolutionChangedEvent>();
```

Comment:
This is the same implementation as `camera_android`, and it doesn't seem to be actually used in any of the federated packages.